apps/calendar/src/controller/column.ts

```diff
@@ -168,13 +183,36 @@ function getRenderInfoOptions(
   };
 }
 
-function setRenderInfo(
-  uiModel: EventUIModel,
-  columnIndex: number,
-  baseWidth: number,
-  startColumnTime: TZDate,
-  endColumnTime: TZDate
-) {
+function setRenderInfo({
+  uiModel,
+  columnIndex,
+  baseWidth,
+  startColumnTime,
+  endColumnTime,
+  isDuplicateEvent = false,
+}: {
+  uiModel: EventUIModel;
+  columnIndex: number;
+  baseWidth: number;
+  startColumnTime: TZDate;
+  endColumnTime: TZDate;
+  isDuplicateEvent?: boolean;
+}) {
+  if (!isDuplicateEvent && uiModel.duplicateEvents.length > 0) {
+    uiModel.duplicateEvents.forEach((event) => {
+      setRenderInfo({
+        uiModel: event,
+        columnIndex,
+        baseWidth,
+        startColumnTime,
+        endColumnTime,
+        isDuplicateEvent: true,
+      });
+    });
+
+    return;
+  }
```

Comment:
얼리리턴보다
```suggestion
  if (isDuplicateEvent || uiModel.duplicateEvents.length === 0) {
    // 아래 로직
  } else {
    uiModel.duplicateEvents.forEach((event) => {
      setRenderInfo({
        uiModel: event,
        columnIndex,
        baseWidth,
        startColumnTime,
        endColumnTime,
        isDuplicateEvent: true,
      });
    });
  }
```
같은 방식은 어떠신가요?

Reply:
저는 개인적으로 얼리 리턴을 더 선호합니다..ㅎㅎㅎ 굳이 else 문을 만들어서 인덴테이션 한 단계를 추가하고 싶지 않기도 하고, 아래 로직에서도 return하는 로직이 없어서 어느 로직을 타더라도 리턴 타입이 동일하게 `undefined`이 되거든요.

Reply:
혹시 else 쓰는 걸 선호하는 이유가 있나요??

Reply:
지금 함수는 괜찮은데 함수가 길어지고 로직이 복잡할 때 중간에 얼리리턴으로 빠져나가면 로직 파악이 쉽지 않더라구요.

개인적인 의견이라 반영 안하셔도 괜찮습니다!

Reply:
저도 기본적으론 얼리리턴 좋아하는데 아무것도 리턴 안하고 `return` 만 있는 얼리리턴보단 값이 명시적인 얼리리턴이어야 그럴만한 가치가 있다고 생각하는 편이에요.

이 함수는 약간 다른데, 재귀함수이므로 조건 종료 부분이 먼저 함수 상단에 드러나있는게 좋다고 생각해서 지금 상태로도 충분하다 봅니다.

제 코멘트에도 썼지만 이런 형태의 재귀가 아닌 다른 방식으로 풀어나갈 수 있으면 좋겠지만 마땅한 개선 아이디어가 떠오르지 않아 넘어가시죠.